app/views/about/index.html.erb

```diff
@@ -31,6 +31,66 @@
     </div>
   </div>
   <hr>
+  <% content_for :title, "CircuitVerse - Core Members" %>
+<div class="container">
+   <div class="row center-row">
+      <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
+         <h2 class="main-heading">CORE TEAM</h2>
+         <p class="main-description">CircuitVerse is lead by an experienced team that is dedicated to innovation and excellence</p>
```

Comment:
Could you add I18n for all newly introduced texts ?

Reply:
@tachyons Yes, I would like to do it, but one thing is can you tell me that in which language I should add the translations